MarkBind/markbind
# Migrate Panel docs

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Other, migration



Part of #274 



**What is the rationale for this request?**


**What changes did you make? (Give an overview)**
1. Removed `Accordion` section, added **Nested Panel** section
2. Changed styling of `Example` and `Markup` containers from the original `vue-strap` guide.

**Is there anything you'd like reviewers to focus on?**
- New **Nested Panel** section.

**Testing instructions:**
1. Checkout **migrate-panel-docs** branch.
2. Run `markbind serve docs` inside the repository folder.
3. Go to **Components Reference** link, check that the included document is rendered correctly.
